src/http/server.ts

```diff
@@ -373,7 +373,7 @@ export class Server {
 
       const accept = request.headers.get("accept") ?? "";
       const contentType = response.headers.get("content-type") ?? "";
-      if (accept.includes("*/*") === false) {
+      if (accept && accept.trim() != "" && accept.includes("*/*") === false) {
```

Comment:
God damn this conditions...

`accept` will always be defined because of line 374, therefore checking for it being defined is not required.

Also, `accept.trim()` should be compared with `!==` not `!=`.

Solutions for this condition may be:
```ts
// Solution 1
if (accept.trim() !== "" && accept.includes("*/*") === false) {
  // Stuff
}

// Solution 2
if (accept.trim().length !== 0 && accept.includes("*/*") === false) {
  // Stuff
}
```

Reply:
OHHHHH wow. i didn't see the `?? ""` part. that def changes things

Reply:
how's this @Guergeiro ? https://github.com/drashland/drash/pull/638/commits/95a419bdf5a60ecff892c9208a6092a27e8dd1d1